README.md

```diff
@@ -40,6 +40,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+Note: maintainers see the [separate docs](maintainers.md)
```

Comment:
nit: suggest changing to...

```
Note: maintainers should refer to the [maintainers guide](maintainers.md)
```

Reply:
updated - thanks

Reply:
(it makes the link a little more accessible to screen readers if the link text description is stronger)